administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+JHtml::_('behavior.formvalidator');
+JHtml::_('behavior.keepalive');
+JHtml::_('formbehavior.chosen', 'select');
+
+$app = JFactory::getApplication();
+$input = $app->input;
+$assoc = JLanguageAssociations::isEnabled();
+
+$app->getDocument()->addScriptDeclaration("
+    function iframeRef( frameRef ) {
+    return frameRef.contentWindow
+        ? frameRef.contentWindow.document
+        : frameRef.contentDocument
+    }
+
+    function triggerSave() {
+        var inside = iframeRef( document.getElementById('target-association') );
+        inside.getElementById('applyBtn').click();
```

Comment:
Actually that click() is [Vanilla JS](http://vanilla-js.com/), didn't use JQuery on that one.